### PR TITLE
Knex seeding

### PR DESCRIPTION
**Pull request by allishultes**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      
Related issues:

- https://github.com/bbc/digital-paper-edit-api/issues/15
- https://github.com/bbc/digital-paper-edit-api/issues/6

**Describe what the PR does**    
Adds seed files to populate DB with mock data. 

To run:
`knex seed:run`.

**State whether the PR is ready for review or whether it needs extra work**    
Four files left to go. Just making sure approach is consistent with ongoing work on database-spike branch. 

**Additional context**    
Branch created from `database-spike` branch.

### CODE REVIEW

**Comment by allishultes** [comment]
Changed this from a draft to ready to review.

File: knex/seeds/05_annotations.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+exports.seed = (knex) => {
+  const dropFk = 'ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk0"; ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk1"; ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk2";';
```
Comment (jamesdools) [inline knex/seeds/05_annotations.js]: if you put backticks you could spread this over a few lines and make it more readable 👍

File: sql/createSchema.sql [hunk 2]
```diff
@@ -73,7 +73,7 @@ CREATE TABLE IF NOT EXISTS "Annotations" (
 CREATE TABLE IF NOT EXISTS "Paper_edits" (
 	"id" serial NOT NULL UNIQUE,
 	"title" TEXT NOT NULL,
-	"desctiption" TEXT NOT NULL,
+	"description" TEXT NOT NULL,
```
Comment (jamesdools) [inline sql/createSchema.sql]: lol

File: knex/knex.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const environment = process.env.MODE_ENV || 'development'
```
Comment (jamesdools) [inline knex/knex.js]: do we need this file? Or could this be incorporated into the db file / main knex instance?

Comment (allishultes) [reply knex/knex.js]: Not sure! Just followed the instructions for creating seed and migration files.

File: knex/seeds/01_users.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+exports.seed = (knex) => {
+  const foreignKeys = 'ALTER TABLE "users_projects" DROP CONSTRAINT IF EXISTS "users_projects_fk0"; ALTER TABLE "Annotations" DROP CONSTRAINT IF EXISTS "Annotations_fk0"; ALTER TABLE "users_transcripts" DROP CONSTRAINT IF EXISTS "users_transcripts_fk0";';
```
Comment (jamesdools) [inline knex/seeds/01_users.js]: like mentioned before - backticks so this could go over a few lines would be good. Shame the `dropForeign` has gone?

**Pull request merged**